addOns/pscanrulesAlpha/src/main/java/org/zaproxy/zap/extension/pscanrulesAlpha/UserControlledOpenRedirectScanner.java

```diff
@@ -41,6 +43,8 @@
 
     private PassiveScanThread parent = null;
 
+    private static final Logger logger = Logger.getLogger(UserControlledOpenRedirectScanner.class);
```

Comment:
This constant should be named in full caps `LOGGER`

Reply:
Updated code. resolving this conversation.